### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by markmandel**
* OpenCensus Link
* Bold for Agones header.
* Backported some CSS classes for the landing page
* Fixed layout of ascicinema
* Switched out incline styles with bootstrap classes.
* Updated the post-css installation to latest.

### CODE REVIEW

**Comment by Kalaiselvi84** [comment]
Everything looks great 😊 
Just a small observation: The alignment for the Agones description and the asciinema player on the front page seems a bit off. Here are screenshots for reference:
Laptop screen: https://screenshot.googleplex.com/8xodM7Pbiy3ZGjC. 
Larger screen: https://screenshot.googleplex.com/4fXFiphXYXZPAL4

File: build/includes/website.mk [hunk 1]
```diff
@@ -36,13 +36,10 @@ site-server: ensure-build-image
 site-static: ensure-build-image
 	-docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) rm -r ./public
 	-mkdir $(agones_path)/site/public
-	# for some reason, this only work locally
-	# postcss-cli@8.3.1 broke things, so pinning the version
 	docker run --rm $(common_mounts) --workdir=$(mount_path)/site $(DOCKER_RUN_ARGS) $(build_tag) \
-		bash -c "npm list postcss-cli || npm install postcss-cli@8.3.0"
-	# autoprefixer 10.0.0 broke things, so pinning the version
+		bash -c "npm list postcss || npm install -D postcss-cli"
```
Comment (Kalaiselvi84) [inline build/includes/website.mk:40]: May be a silly question. The latest version is now stable for our use?

Comment (markmandel) [reply build/includes/website.mk:40]: I tired it - it worked 🤷🏻 😄

**Comment by markmandel** [comment]
>Just a small observation: The alignment for the Agones description and the asciinema player on the front page seems a bit off

Good catch, I'll take a look.

**Comment by markmandel** [comment]
That should fix it (or at least as good as it's going to get I think).

**Comment by Kalaiselvi84** [comment]
I've observed that the change didn't completely resolve the problem. Here are the current alignments on different screens:

Small screen - https://screenshot.googleplex.com/5TgLCDhEuYchhiQ
Larger screen - https://screenshot.googleplex.com/BMcPcL4Mohsrqmq

From what I've noticed:

Col-4 classes, such as `<div class="col col-4">, <div class="col col-sm-4">, <div class="col col-lg-4">, and <div class="col col-md-4">`, display properly on small screens.
Similarly, col-3 classes like `<div class="col col-3">, <div class="col col-lg-3">, and <div class="col col-md-3">` align well on larger screens.

However, when there's a combination of col-4 and col-3, it seems to create alignment issues on smaller screens.

**Comment by markmandel** [comment]
Yeah that's a good point.

I used a `col-lg-4` and combo'd it with some custom css to set the max width on larger screens.

PTAL, lemme know what you think!

File: site/assets/scss/_variables_project.scss [hunk 2]
```diff
@@ -46,6 +48,16 @@ $td-sidebar-tree-root-color: #222 !default;
     background-color:black !important;
 }
 
+.showcase img {
+    margin: 0 30px;
+}
+
+#cover-title {
+    max-width: 800px;
```
Comment (Kalaiselvi84) [inline site/assets/scss/_variables_project.scss:56]: I think 500px will work for both small and larger screens.

Small screen - https://screenshot.googleplex.com/6QUiQJ96FwK7qwk

Larger screen - https://screenshot.googleplex.com/9udSepQgZfV2Jz4

Comment (markmandel) [reply site/assets/scss/_variables_project.scss:56]: I liked the responsiveness as the width changes (so a bit wider on wider screens), but at this point, we're just down to personal preference I think.

Comment (Kalaiselvi84) [reply site/assets/scss/_variables_project.scss:56]: Thanks for sharing your thoughts! I value your input...

Would it be appropriate to consolidate the changes from both https://github.com/googleforgames/agones/pull/3382 and https://github.com/Kalaiselvi84/agones/pull/325 into https://github.com/googleforgames/agones/pull/3415? The latest PR contains the accurate go.mod and go.sum files.

Comment (markmandel) [reply site/assets/scss/_variables_project.scss:56]: Yeah - if you want to take these changes, and whatever other changes you think were appropriate and build out a new PR, that works 👍🏻 and we can do a final review.

**Pull request closed without merging**